File: app/src/main/java/com/italankin/dictionary/ui/settings/SettingsSearchOptionsFragment.java
package com.italankin.dictionary.ui.settings;

import android.os.Bundle;
import android.preference.PreferenceFragment;

import com.italankin.dictionary.R;

/**
 * Additional fragment for settings screen. Displays search options.
 */
public class SettingsSearchOptionsFragment extends PreferenceFragment {

    @Override
    public void onCreate(final Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setRetainInstance(true);
        addPreferencesFromResource(R.xml.prefs_search);
    }

}

File: app/src/main/java/com/italankin/dictionary/ui/settings/SettingsSearchOptionsFragment.kt
package com.italankin.dictionary.ui.settings

import android.os.Bundle
import androidx.preference.PreferenceFragmentCompat
import com.italankin.dictionary.R

/**
 * Additional fragment for settings screen. Displays search options.
 */
class SettingsSearchOptionsFragment : PreferenceFragmentCompat() {

    override fun onCreatePreferences(savedInstanceState: Bundle, rootKey: String) {
        addPreferencesFromResource(R.xml.prefs_search)
    }
}
